Update README.md (#2)
vtex-apps/onesignal: docs/README.md

[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->

> OneSignal first party integration app. The [solution](https://onesignal.com/) mainly works with web push notifications and in-app messages for customer engagement.

Reviewer: ```suggestion
OneSignal first party integration app. The [solution](https://onesignal.com/) provides web push notifications for customer engagement.
```
